src/migrations/1696994943309-bloom-backend.ts

```diff
@@ -0,0 +1,78 @@
+        await queryRunner.query(`ALTER TABLE "course_partner" DROP CONSTRAINT "FK_4799c98d625acaae457c7dd23da"`);
+        await queryRunner.query(`ALTER TABLE "course_partner" DROP CONSTRAINT "FK_96575d601e92c89fa881d2eb128"`);
+        await queryRunner.query(`ALTER TABLE "course_user" DROP CONSTRAINT "FK_70824fef35e6038e459e58e0358"`);
+        await queryRunner.query(`ALTER TABLE "course_user" DROP CONSTRAINT "FK_062e03d78da22a7bd9becbfaaac"`);
+        await queryRunner.query(`ALTER TABLE "subscription_user" DROP CONSTRAINT "FK_0feee04ab572b223d511672be81"`);
+        await queryRunner.query(`ALTER TABLE "subscription_user" DROP CONSTRAINT "FK_cdf67f6c499d7a4c7b4d1524850"`);
+        await queryRunner.query(`ALTER TABLE "partner_access" DROP CONSTRAINT "FK_e25f0b4b2c6fbddc8375b02f73e"`);
+        await queryRunner.query(`ALTER TABLE "partner_access" DROP CONSTRAINT "FK_60f11a3686ca313f1ac25f689f9"`);
+        await queryRunner.query(`ALTER TABLE "partner_access" DROP CONSTRAINT "FK_25e7860a2eec1f9366fcfe3a954"`);
+        await queryRunner.query(`ALTER TABLE "therapy_session" DROP CONSTRAINT "FK_58700a8a5d47651d0e895a86b4e"`);
+        await queryRunner.query(`ALTER TABLE "therapy_session" DROP CONSTRAINT "FK_95b7041e4d05e914cde6b3753d0"`);
+        await queryRunner.query(`ALTER TABLE "partner_admin" DROP CONSTRAINT "FK_c7ee0521b73218dd1f0b13e23d5"`);
+        await queryRunner.query(`ALTER TABLE "partner_admin" DROP CONSTRAINT "FK_825f3ea183aebdb95a52f1f972c"`);
+        await queryRunner.query(`ALTER TABLE "partner_feature" DROP CONSTRAINT "FK_c824581c685cf5969f43732ec75"`);
+        await queryRunner.query(`ALTER TABLE "partner_feature" DROP CONSTRAINT "FK_91fa49f0f077d1d47ef581d458c"`);
+        await queryRunner.query(`ALTER TABLE "session_user" DROP CONSTRAINT "FK_3014902f31f2a83ec475d75bec8"`);
+        await queryRunner.query(`ALTER TABLE "session_user" DROP CONSTRAINT "FK_5452e53b773936e51ff0e96d064"`);
+        await queryRunner.query(`ALTER TABLE "session" DROP CONSTRAINT "FK_2440b236e81d633ff0613ae59d4"`);
+        await queryRunner.query(`DROP TABLE "email_campaign"`);
+        await queryRunner.query(`DROP TABLE "course_partner"`);
+        await queryRunner.query(`DROP TABLE "course"`);
+        await queryRunner.query(`DROP TABLE "course_user"`);
+        await queryRunner.query(`DROP TABLE "user"`);
+        await queryRunner.query(`DROP TABLE "subscription_user"`);
+        await queryRunner.query(`DROP TABLE "subscription"`);
+        await queryRunner.query(`DROP TABLE "partner_access"`);
+        await queryRunner.query(`DROP TABLE "therapy_session"`);
+        await queryRunner.query(`DROP TABLE "partner_admin"`);
+        await queryRunner.query(`DROP TABLE "partner"`);
+        await queryRunner.query(`DROP TABLE "partner_feature"`);
+        await queryRunner.query(`DROP TABLE "feature"`);
+        await queryRunner.query(`DROP TABLE "session_user"`);
+        await queryRunner.query(`DROP TABLE "session"`);
+    }
```

Comment:
Hi @jobayer12, great work, thank you! For some reason it looks like the migration is picking up the entire database. Not sure why! My guess is that you should:
-  comment out the entity changes
- revert the migration
- try running the backend without your changes
- Put back your entity changes
- Run the generate:migration command in the package.json
This should hopefully give you the right migration file. Let me know if that works or not!

Reply:
@eleanorreem Thanks for your instructions. I figured out where is the issue and resolved it. Review my pull again.